Mathlib/ModelTheory/Order.lean

```diff
@@ -238,6 +250,14 @@ theorem realize_noTopOrder [h : NoTopOrder M] : M ⊨ L.noTopOrderSentence :=
 theorem realize_noBotOrder [h : NoBotOrder M] : M ⊨ L.noBotOrderSentence :=
   realize_noBotOrder_iff.2 h
 
+theorem noTopOrder_of_dlo [h: M ⊨ L.dlo] : NoTopOrder M :=
```

Comment:
```suggestion
theorem noTopOrder_of_dlo [h : M ⊨ L.dlo] : NoTopOrder M :=
```
or maybe
```suggestion
theorem noTopOrder_of_dlo [M ⊨ L.dlo] : NoTopOrder M :=
```
? Why are you naming a typeclass argument?

Reply:
Because I was copy-pasting from the theorems right above a little too lazily.